src/utils/database.js

```diff
@@ -3,19 +3,17 @@
 const mongoose = require('mongoose');
 const diffHistory = require('mongoose-diff-history/diffHistory');
 const uriFormat = require('mongodb-uri');
+const { Configuration } = require('@schul-cloud/commons');
 
 const logger = require('../logger');
 const {
 	NODE_ENV,
-	DATABASE_AUDIT,
 	MONGOOSE_CONNECTION_POOL_SIZE,
-	DB_URL,
-	DB_USERNAME,
-	DB_PASSWORD,
 	ENVIRONMENTS,
 } = require('../../config/globals');
 
-if (DATABASE_AUDIT === 'true') {
+
+if (Configuration.get('DATABASE__AUDIT') === 'true') {
```

Comment:
when using get, mke this field required in schema too like url or here and below ask for Configuration.has too